draft-ietf-httpbis-priority.md

```diff
@@ -309,6 +312,26 @@ properties of an HTTP request it receives, the server is expected to control the
 cacheability or the applicability of the cached response, by using header fields
 that control the caching behavior (e.g., Cache-Control, Vary).
 
+# Signalling Initial Priority in Frames
+
+This document specifies a new PRIORITY_UPDATE frame for HTTP/2 ({{!RFC7540}})
+and HTTP/3 ({{!I-D.ietf-quic-http}}). It carries priority parameters and
+references the target of the prioritization based on a version-specific
+identifier; in HTTP/2 this is the Stream ID, in HTTP/3 this is either the Stream
+ID or Push ID.
+
+PRIORITY_UPDATE can be sent by clients on the control stream. This allows the
+PRIORITY_UPDATE to be sent before the stream it references is created, and
+avoids having to extend the protocol semantics to support continued updates
+during the stream lifetime; see {{reprioritization}}.
+
+Unlike the Priority header field, the PRIORITY_UPDATE frame is a hop-by-hop
+signal. Clients that do not want to expose an end-to-end signal MAY omit the
```

Comment:
I just have to say that this "do not want" statement assumes a great deal without a whole lot of support.

Reply:
paging @kazuho who suggested this text

Reply:
should be fixed in https://github.com/httpwg/http-extensions/pull/1167/commits/ca71a499138f646c99fc1c57a72277f2db4fc1e0